app/(public)/tailwind/plugins/components/typography/sul-typography.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * SUL specific typography styles
+ */
+module.exports = function () {
+    return function ({ addComponents }) {
+      const components = {
+        'h1, h2, h3, h4, h5': {
+          fontFamily: '"Source Serif Pro", Georgia, Times, "Times New Roman", serif',
+        },
+        'h2': {
+          letterSpacing: '-0.005em',
+        },
+        'h3': {
+          letterSpacing: '-0.003em',
+        },
```

Comment:
@jdwjdwjdw I think these styles belong more in a base.js file not as TW components 😄

Reply:
For example, on MyAccount, we add a project base.js file which includes some link style overrides
https://github.com/SU-SWS/adapt-my-account/blob/dev/src/tailwind/plugins/base/base.js